### ISSUE TITLE
Elevated circulating sitosterol concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6470]
**Preferred term label:**
Elevated circulating sitosterol concentration

**Synonyms**
Elevated circulating sitosterol concentration
Elevated plasma beta-sitosterol

**Definition (free text, please give PubMed ID)**
An increased concentration of beta-sitosterol in the blood circulation.
Comment: sitosterol (also known as beta-sitosterol) is a phytosterols with a chemical structure similar to that of cholesterol. 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0003119

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sitosterolemia 1 and 2
https://pubmed.ncbi.nlm.nih.gov/11138003/
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
also https://pubmed.ncbi.nlm.nih.gov/20719861/ 1/1 Elevated circulating sitosterol concentration

**Comment by pnrobinson** [comment]
added
